pkg/protoutils/bench_test.go

```diff
@@ -3,6 +3,7 @@ package protoutils
 import (
 	"testing"
 
+	_ "github.com/gogo/googleapis/google/api"
```

Comment:
I don't quite get why do we need this line, @dmvolod could you please elaborate?